operations/toc-chair.md

```diff
@@ -0,0 +1,40 @@
+# TOC Chair
+
+This document serves to provide the TOC Chair with details on their responsibilities, role, and expectations that are in addition to those of a TOC member.
+
+## Selecting the Chair
+
+The TOC Chair is an member of the TOC that is chosen by each seated TOC to lead and presented the TOC in the execution of it's work. 
+
+## Functions of the Chair
```

Comment:
Maybe we can provide a guideline on this: It sounds like the TOC chair role requires additional time commitment to fulfill the functions in addition to the time commitment stipulated for a non chair TOC member.  Is it more like 25% instead of 10-25%?

Reply:
good call out - adding